utils/run_in_ga.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+SCONS=scons
+
+if [ ! -e /usr/bin/scons ]
+then
+    SCONS=scons-3
+fi
+
+echo ::group::Rebuild spdk
+rm -rf /opt/daos/prereq/release/spdk
+$SCONS PREFIX=/opt/daos --build-deps=yes --deps-only
+echo ::endgroup::
+
+echo ::group::Build type debug.
+$SCONS --jobs 10 PREFIX=/opt/daos COMPILER=$COMPILER TARGET_TYPE=release BUILD_TYPE=debug
```

Comment:
(style)  line over 80 characters
(lint) Double quote to prevent globbing and word splitting. [SC2086]